crypto/src/openssl.rs

```diff
@@ -184,24 +184,14 @@ impl Crypto for OpensslCrypto {
 
         Ok(EcdsaPub { x, y })
     }
-
+    
```

Comment:
It looks like you might need to run `cargo fmt`. It should get rid of extra whitespace. You can run the CI tests locally with the `ci.sh` script at the top level. The script will remind you run `cargo fmt` if you need to.

Reply:
Done.